core/sugarTraversals.ml

```diff
@@ -795,7 +807,7 @@ class map =
       | `Type t -> `Type (o#typ t)
       | `Row r -> `Row (o#type_row r)
       | `Presence p -> `Presence (o#type_field_spec p)
-    method tyvar : tyvar -> tyvar = o#unknown
+    (* method tyvar : tyvar -> tyvar = o#unknown *)
```

Comment:
Is there a reason for this to remain but be commented out?

Reply:
removed